core/snippets/snippets/markdown.snippet

```diff
@@ -3,12 +3,15 @@ language: markdown
 
 name: link
 phrase: link
+
+$1.wrapperPhrase: link
 -
 [$1]($0)
 ---
 
 name: image
 phrase: image
+$1.wrapperPhrase: image
```

Comment:
The choice for both image and link is to focus on wrapping the text with a link. We could add a wrapper phrase for `$0` but I could not find a good name for it, and that flow feels less natural since what is likely to be dictated first is the display text, not the link URL.